### ISSUE TITLE
Angular 9.1 and Transloco-locale 1.1.2 giving injection error

### ISSUE THREAD

**Issue by Siedlerchr** [ngneat/transloco#267]
## I'm submitting a...

<pre><code>
[ ] Regression (a behavior that used to work and stopped working in a new release)
[x] Bug report  
[ ] Performance issue
[ ] Feature request
[ ] Documentation issue or request
[ ] Support request
[ ] Other... Please describe:
</code></pre>

## Current behavior

Update to transloco-locale 1.1.2 and notice that the error is visible:
Previous version 1.1.1 worked fine.  

Firefox:
```
 "R3InjectorError(MyComponentModule)[TranslocoLocaleService -> TranslocoLocaleService -> TranslocoLocaleService -> TranslocoService -> InjectionToken TRANSLOCO_TRANSPILER -> InjectionToken TRANSLOCO_TRANSPILER -> InjectionToken TRANSLOCO_TRANSPILER]: 
  NullInjectorError: No provider for InjectionToken TRANSLOCO_TRANSPILER!"

```
Chrome:
```
core.js:6185 ERROR Error: NullInjectorError: No provider for InjectionToken TRANSLOCO_TRANSPILER!
    at NullInjector.get (core.js:1076) [angular]
    at R3Injector.get (core.js:16629) [angular]
    at R3Injector.get (core.js:16629) [angular]
    at R3Injector.get (core.js:16629) [angular]
    at injectInjectorOnly (core.js:931) [angular]
    at Module.ɵɵinject (core.js:941) [angular]
    at Object.TranslocoService_Factory [as factory] (ngneat-transloco.js:1302) [angular]
    at R3Injector.hydrate (core.js:16865) [angular]
    at R3Injector.get (core.js:16617) [angular]
    at injectInjectorOnly (core.js:931) [angular]
    at ɵɵinject (core.js:941) [angular]
    at Object.TranslocoLocaleService_Factory [as factory] (ngneat-transloco-locale.js:844) [angular]
    at R3Injector.hydrate (core.js:16865) [angular]
    at R3Injector.get (core.js:16617) [angular]
defaultErrorLogger @ core.js:6185
6core.js:6185 ERROR Error: Cannot instantiate cyclic dependency! TranslocoLocaleService
    at throwCyclicDependencyError (core.js:8035) [angular]
    at R3Injector.hydrate (core.js:16861) [angular]
    at R3Injector.get (core.js:16617) [angular]
    at NgModuleRef$1.get (core.js:36024) [angular]
    at R3Injector.get (core.js:16629) [angular]
    at NgModuleRef$1.get (core.js:36024) [angular]
    at R3Injector.get (core.js:16629) [angular]
    at NgModuleRef$1.get (core.js:36024) [angular]
    at Object.get (core.js:33773) [angular]
    at getOrCreateInjectable (core.js:5805) [angular]
    at Module.ɵɵdirectiveInject (core.js:20861) [angular]
    at TranslocoDatePipe_Factory (ngneat-transloco-locale.js:941) [angular]
    at Module.ɵɵpipe (core.js:36605) [angular]
    at MyComponent_ng_container_0_ng_template_37_Template (my.component.html:64) [angular]

```

## Expected behavior
No errors.

## Minimal reproduction of the problem with instructions

Have both transloco and transloco.-locale installed and configured as providers:
app  Module:
```
export const globalFormatConfig: LocaleFormatOptions = {
  date: {
    dateStyle: 'medium',
    year: 'numeric',
    month: '2-digit',
    day: '2-digit',
    timeStyle: 'medium',
    hour: '2-digit',
    minute: '2-digit',
    second: '2-digit',
  }
};

imports: [
 TranslocoLocaleModule.init({
      localeConfig: {
        global: globalFormatConfig,
        localeBased: {
          'de-DE': globalFormatConfig,
          'en-US': globalFormatConfig
        }
      }
    }),
    TranslocoModule,
].
providers: [
  {
      provide: TRANSLOCO_CONFIG,
      useValue: {
        availableLangs: ['en-US', 'de-DE'],
        defaultLang: 'de-DE',
        fallbackLang: 'en-US',
        prodMode: environment.production,
      } as TranslocoConfig
    },
    TranslocoDatePipe,
    translocoLoader,
}
]
```

In MyComponent Module:
```
imports: [
        TranslocoLocaleModule,
        TranslocoModule,
]
```

For bug reports please provide the _STEPS TO REPRODUCE_ and if possible a _MINIMAL DEMO_ of the problem, for that you could use our [stackblitz example](https://stackblitz.com/edit/transloco-example)

## Environment

Angular version: 9.1.0

Browser:
- [x] Chrome (desktop) version 80
- [ ] Chrome (Android) version XX
- [ ] Chrome (iOS) version XX
- [ ] Firefox version 75
- [ ] Safari (desktop) version XX
- [ ] Safari (iOS) version XX
- [ ] IE version XX
- [ ] Edge version XX
 
For Tooling issues:

- Node version: v12.16.1
- Platform:  Windows

Others:
Angular 9.1.0, Ionic 5
```
   "@angular/common": "~9.1.0",
    "@angular/compiler": "~9.1.0",
    "@angular/core": "~9.1.0",
    "@angular/forms": "~9.1.0",
    "@angular/platform-browser": "~9.1.0",
    "@angular/platform-browser-dynamic": "~9.1.0",
    "@angular/router": "~9.1.0",
   "@ngneat/transloco": "^2.14.2",
    "@ngneat/transloco-locale": "^1.1.2",
```

**Comment by Ketec** [comment]
Same - wanted to upgrade to 1.1.2 hoping it fixes the unit test pipe errors, but getting the same injection errors.
Some testing showed that the error shows up when locale service is used (I disabled all submodules, main app just used the service to set active locale).

**Comment by shaharkazaz** [comment]
@Ketec @Siedlerchr can you try v1.2.0?

**Comment by Siedlerchr** [comment]
For me it's fixed with 1.2.0
